docs/developers/bitesize/workflowExecution/executionAndValueStoreExamples.md

```diff
@@ -0,0 +1,229 @@
+
+The start of workflow execution looks like this:
+
+| | `x` | `ScatterNode` | `results_count` |
+|---|---|---|---|
+|1|`NotStarted`|`NotStarted`| `NotStarted` |
+|2|`Running`|`NotStarted`| `NotStarted` | 
+|3|`Done`|`NotStarted`| `NotStarted` | 
+
+Once `x` is evaluated the value store gains an entry:
+```json
+{
+  "x": "[0, 1]"
+}
+```
+
+The scatter node now becomes runnable because its upstream dependency (`x`) is `Done` in the Execution Store.
+
+The evaluation of `ScatterNode` updates the execution store in a number of ways:
+
+* One call key for each index of `scattered_task` is added.
+* The `scattered_task` gets an un-indexed key too. This key is used to mark when all of the shards of the call are complete.
+* The gathered value `scattered_task.string_out` represents the "gathered" results of the task's output. It only runs 
+once the un-indexed `scattered_task` key is Done and gathers output values into an array.
+This gather key also acts as the upstream dependency of the `results_count` output expression.
+* The `ScatterNode` is marked as `Done` so that it doesn't get triggered to run again.
+
+Following the scatter-expansion evaluation of `ScatterNode`, the Execution Store looks like this:
+
+| | `x` | `ScatterNode` | `scattered_task:0` | `scattered_task:1` | `scattered_task` | `scattered_task.string_out` | `results_count` |
+|---|---|---|---|---|---|---|---|
+|4|`Done`|`Done`|`NotStarted`|`NotStarted`|`NotStarted`|`NotStarted`|`NotStarted`
+
+The Value Store is not changed at this time because no new values have been generated.
+
+#### Parallel Shard Execution
+
+The two scattered shards are now immediately runnable because they have no upsteam dependencies.
+As the two jobs are run, the Execution Store map updates to track their statuses:
+
+| | `x` | `ScatterNode` | `scattered_task:0` | `scattered_task:1` | `scattered_task` | `scattered_task.string_out` | `results_count` |
+|---|---|---|---|---|---|---|---|
+|4|`Done`|`Done`|`NotStarted`|`NotStarted`|`NotStarted`|`NotStarted`|`NotStarted`
+|5|`Done`|`Done`|`QueuedInCromwell`|`NotStarted`|`NotStarted`|`NotStarted`|`NotStarted`
+|6|`Done`|`Done`|`QueuedInCromwell`|`QueuedInCromwell`|`NotStarted`|`NotStarted`|`NotStarted`
+|7|`Done`|`Done`|`Starting`|`QueuedInCromwell`|`NotStarted`|`NotStarted`|`NotStarted`
+|8|`Done`|`Done`|`Starting`|`Starting`|`NotStarted`|`NotStarted`|`NotStarted`
+|9|`Done`|`Done`|`Running`|`Starting`|`NotStarted`|`NotStarted`|`NotStarted`
+|10|`Done`|`Done`|`Running`|`Running`|`NotStarted`|`NotStarted`|`NotStarted`
+|11|`Done`|`Done`|`Running`|`Done`|`NotStarted`|`NotStarted`|`NotStarted`
+|12|`Done`|`Done`|`Done`|`Done`|`NotStarted`|`NotStarted`|`NotStarted`
+
+As the results for each shard come in, the value store is also updated to include them:
+
+At step 11 (shard 1 has finished but shard 0 has not):
+```json
+{
+  "x": "[0, 1, 2]",
```

Comment:
The value of `x` here and everywhere below should be `[0, 1]`

Reply:
Thanks! I made the workflow simpler to produce less rows in the tables, but forgot to update this part 🤦‍♂️